src/main/java/hunternif/mc/impl/atlas/client/ClientModEvents.java

```diff
@@ -0,0 +1,57 @@
+package hunternif.mc.impl.atlas.client;
+
+import hunternif.mc.impl.atlas.AntiqueAtlasMod;
+import hunternif.mc.impl.atlas.ClientProxy;
+import hunternif.mc.impl.atlas.marker.MarkerTextureConfig;
+import hunternif.mc.impl.atlas.registry.MarkerType;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.RegisterClientReloadListenersEvent;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+
+@Mod.EventBusSubscriber(modid = AntiqueAtlasMod.ID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
+public class ClientModEvents {
```

Comment:
I think this class shouldn't exists. Please move that stuff into `AntiqueAtlasModClient`.
I'm trying to minimize the differences between the fabric and forge code.